app/importers/concerns/importable.rb

```diff
@@ -172,6 +172,7 @@ def process_data_row(attributes, index, last_attempt: true)
       register_result(index, class: record.class.name, state: :failure, message: error_message, errors: errors)
       nil
     rescue StandardError => e
+      raise Importo::RetryError.new("ActiveRecord::RecordNotUnique",5) if !last_attempt && e.is_a?(ActiveRecord::RecordNotUnique)
```

Comment:
looks good, but not sure about the logic behind it. may be move this part to the concerned importer, it will not be a good idea to retry all importers for this exception